.github/workflows/quality.yml

```
@@ -0,0 +1,65 @@
+        if [ -n "$(gofmt -l .)" ]; then
+          echo "The following files are not properly formatted:"
+          gofmt -l .
+          exit 1
+        fi
+        
+    - name: Install golangci-lint
+      uses: golangci/golangci-lint-action@v6
+      with:
+        args: --timeout=5m --modules-download-mode=mod
+        version: v1.60.3
+        args: --timeout=5m
```

actionlint (1.7.7)

key "args" is duplicated in "with" section. previously defined at line:44,col:9. note that this key is case insensitive

(syntax-check)